### PR TITLE
add sdkman tip

### PR DESCRIPTION
**Pull request by varasu**
Signed-off-by: Veselin Vlasakiev <veselin@veselin.org>

### CODE REVIEW

File: docs/getting-started/prerequisites.md [hunk 1]
```diff
@@ -285,6 +285,10 @@ Alternatively, you can install Helm Diff by following the instructions below.
 
 ### OpenJDK
 
+!!! tip
+    In case you have trouble installing the right jdk version,
```
Comment (strieflin) [inline docs/getting-started/prerequisites.md:289]: Not sure if people are actually having _trouble_ installing the right JDK. I would reformulate this as follows:

> For easy installation and management of parallel JDK versions check [sdkman](https://sdkman.io/).

**Review by strieflin — CHANGES_REQUESTED** [review]
Please add yourself as a contributor to NOTICE.md.

File: docs/getting-started/prerequisites.md (hunk 1 above)
Comment (varasu) [reply docs/getting-started/prerequisites.md:289]: It depends on the distro : )

**Review by strieflin — APPROVED** [review]
LGTM

**Comment by strieflin** [comment]
Thank you @varasu for your first contribution. Welcome to the team!

**Pull request merged**